lte/gateway/c/session_manager/test/Consts.h

```diff
@@ -32,6 +32,8 @@
 #define IPv6_2 "2001:0db8:0a0b:12f0:0000:0000:0000:0002"
 #define IPv6_3 "2001:0db8:0a0b:12f0:0000:0000:0000:0003"
 #define IPv6_4 "2001:0db8:0a0b:12f0:0000:0000:0000:0004"
+#define UE_IPV4 "10.20.30.40"
```

Comment:
Let's match the naming pattern laid out above. UE_IPV6 -> IPv6_5 ? UE_IPV4 -> IPv4_1

Reply:
Modified the code as suggested.